### DISCUSSION TITLE
shortcut to turn "ignore google chrome" on and off

### DISCUSSION THREAD

**Discussion by bpdash** [rxhanson/Rectangle#1507] [Q&A]
I'm looking for a way to quickly turn the "ignore google chrome" option on and off. The use case here is to move windows around, then do some work in google sheets (or something else where shortcuts conflict) and then switch back to navigation without needing to click into the options each time.

I'm working on Mac.

**Comment by rxhanson** [comment]
There's this PR that's being actively worked on for tying a URL to the ignore/unignore behavior: https://github.com/rxhanson/Rectangle/pull/1505

I see no reason to turn down a good follow-on PR for one that ties it to a keyboard shortcut.
